File: baekjoon/9498.js
const fs = require('fs')

const input = fs.readFileSync('/dev/stdin').toString()

switch (Math.floor(Number(input) / 10)) {
  case 9:
  case 10:
    console.log('A')
    return
  case 8:
    console.log('B')
    return
  case 7:
    console.log('C')
    return
  case 6:
    console.log('D')
    return

  default:
    console.log('F')
    return
}

File: baekjoon/단계별로 풀어보기/if문/9498.py
import math
score = math.floor(int(input())/10)
if score == 10 or score == 9:
  print('A')
elif score == 8:
  print('B')
elif score == 7:
  print('C')
elif score == 6:
  print('D')
else:
  print('F')